release/models/openconfig-extensions.yang

```diff
@@ -160,11 +166,17 @@ module openconfig-extensions {
   extension telemetry-atomic {
     description
       "The telemetry-atomic annotation is specified in the context of
-      a subtree (container, or list), and indicates that all nodes
+      a subtree (container, or leaf-list), and indicates that all nodes
       within the subtree are always updated together within the data
       model. For example, all elements under the subtree may be updated
       as a result of a new alarm being raised, or the arrival of a new
-       protocol message.
+      protocol message. When the annotation is specified in the context
+      of a list of containers, it applies to individual list elements
+      (subtree or container and all elements under that subtree) of the
+      list and not to the entire list. For example, say ipv4-entry list
+      has two elements, prefix1 and prefix2, all elements under the
+      prefix1 subtree may be updated as a result of a next-hop group
+      change while prefix2 subtree is not updated.
```

Comment:
Would it be more clear to say something like:
"the atomic property applies to the contents of each individual keyed entry in the list."